src/statements.md

```diff
@@ -75,11 +75,13 @@ except when they are shadowed by another variable declaration.
 >     ( `:` [_Type_] )<sup>?</sup> `=` [_Expression_] `else` [_BlockExpression_] `;`
 
 A *`let else` statement* introduces a new set of [variables], given by a
-refutable [pattern]. The pattern is followed optionally by a type
-annotation. When no type annotation is given, the compiler will infer the type,
-or signal an error if insufficient type information is available for definite
-inference. It is is then followed by an initializer expression as well as a
-diverging `else` block.
+[pattern]. The pattern is followed optionally by a type annotation. When no type
+annotation is given, the compiler will infer the type, or signal an error if
+insufficient type information is available for definite inference. It is is
+then followed by an initializer expression as well as a diverging `else` block.
+Compared to `let` statement patterns which have to be irrefutable, `let else`
+patterns can be both irrefutable and refutable. However, as it is besides the
+point of, irrefutable patterns in `let else` are linted against.
```

Comment:
`as it is besides the point of` reads really strange...

Something like this would be better:
```
However, irrefutable patterns in `let else` are linted against due to their lack of usefulness (the `else` would be dead code).
```

Reply:
What about "Irrefutable patterns are not forbidden, however there is a lint for them as the `else` clause is redundant"?

Reply:
That sounds better.

Reply:
I've applied a slightly different version, but one that avoids the "besides the point" formulation.

Reply:
Looks good to me